api/data_refinery_api/management/commands/post_downloads_summary.py

```diff
@@ -105,14 +105,13 @@ def should_display_email(email: str) -> bool:
     """ Returns true if the given email is not associated with the CCDL suers """
     if not email:
         return False
-
     return not (
         email.startswith("cansav09")
         or email.startswith("arielsvn")
         or email.startswith("jaclyn.n.taroni")
         or email.startswith("kurt.wheeler")
         or email.startswith("greenescientist")
-        or "@alexslemonade.org" not in email
+        or "@alexslemonade.org" in email
```

Comment:
Does this mean we are now including `alexslemonade` emails in the summaries? I was using mine for testing purposes, i must have created 100 datasets this sprint.

Reply:
No, in this case, there was a double negation. Note that there's a `not` also at the beginning of the condition. Should we use a variable to make this easier to see?

Reply:
i would just remove it and add email.endsWith("@alexslemonade.org") after the begins with rules

Reply:
Sure!

```suggestion
        or email.endswith("@alexslemonade.org")
```